charts/drupal/values.schema.json

```diff
@@ -251,6 +251,7 @@
               "command": { "type": "string" },
               "backoffLimit": { "type": "integer" },
               "nodeSelector": { "type": ["object", "null"] },
+              "php_ini": { "type": "string" },
```

Comment:
Please add this new key also to the default drupal cron job in values.yaml, even if it would be just an empty value, but so that people might notice that it is available. A brief comment would also be nice.